### PR TITLE
Modernize

### PR DESCRIPTION
**Pull request by mih**
Just just be a `make` still, but:

- ensure virtual env with proper dependencies and remodnav version
- all imports on top (immediate and complete dependency check)
- no longer piping outputs (did prevent progress bars from showing)
- avoid computing some pieces twice
- consolidate datalad API usage on a single function (`get`)
- some more explicit function names
- some import duplication removed

### CODE REVIEW

File: code/mk_figuresnstats.py [hunk 1]
```diff
@@ -862,13 +866,13 @@ def savegaze():
             show_vels=True,
             coord_lim=(0, 1280),
             vel_lim=(0, 1000))
```
Comment (adswa) [inline code/mk_figuresnstats.py:868]:
```suggestion
            vel_lim=(0.001, 1000))
```

While we are at it, we could set the lower bounds of the y-axis to not exactly zero. The data is logscaled, and this results in 

```
/home/adina/repos/papers/paper-remodnav/remodnav/remodnav/tests/utils.py:124: UserWarning: Attempted to set non-positive bottom ylim on a log-scaled axis.
Invalid limit will be ignored.
  ax2.set_ylim(vel_lim)
```

**Review by adswa — APPROVED** [review]
This looks good, and makes a much nicer terminal output! I added a suggestion to get rid of one remaining warning. We should, however, also update the README accordingly. Currently, it still advises to install the requirements by hand.

**Comment by adswa** [comment]
Oh one more thought: ``remodnav`` doesn't have datalad as a requirement, as far as I remember. Can we add this, or point people towards installing it themselves?

**Comment by adswa** [comment]
I pushed a README update

**Comment by mih** [comment]
Thx!

**Pull request merged**